x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SlmHealthIndicatorServiceTests.java

```diff
@@ -68,7 +71,13 @@ public void testIsYellowWhenNotRunningAndPoliciesConfigured() {
                     YELLOW,
                     "SLM is not running",
                     new SimpleHealthIndicatorDetails(Map.of("slm_status", status, "policies", 1)),
-                    Collections.emptyList(),
+                    Collections.singletonList(
+                        new HealthIndicatorImpact(
+                            3,
+                            "Scheduled snapshots are not running, which could lead to future data loss.",
```

Comment:
Similar to my comment above
```suggestion
                            "Scheduled snapshots are not running.  Enable schedule snapshot management to prevent loss of data in the future."
```

Reply:
Since this is the impacts block rather than user actions, how about `Scheduled snapshots are not running. There might not be backups of the data.`? Or `Scheduled snapshots are not running. There might not be backups of the data that could be used to restore if data is lost in the future.`?

Reply:
I know it's a bit wordy, but this feels more correct than what we had previous 👍 .